src/validators/foreignborndocument.js

```diff
@@ -1,13 +1,25 @@
 import { validateModel } from 'models/validate'
 import foreignBornDocument from 'models/foreignBornDocument'
+import store from 'services/store'
+import * as formTypes from 'constants/formTypes'
+import { requireRelationshipMaritalForeignBornDocExpiration } from 'helpers/branches'
 
-export const validateForeignBornDocument = data => (
-  validateModel(data, foreignBornDocument) === true
-)
+export const validateForeignBornDocument = (data, formType = formTypes.SF86) => {
+  const isForeignBornDocExpirationRequired = requireRelationshipMaritalForeignBornDocExpiration(formType)
+  return validateModel(
+    data,
+    foreignBornDocument,
+    { requireForeignBornDocExpiration: isForeignBornDocExpirationRequired }
```

Comment:
I don't think this option is being passed down correctly from the marital or civil union validator... on the SF-86, the section shows as valid in the sidebar even if there is no value in the foreign born doc expiration date.